### ISSUE TITLE
[Jupiter] Train VGG16 with cross validation on preterm dataset

### ISSUE THREAD

**Issue by Domnyk** [Domnyk/pallas#25]
Assumptions 
* Train VGG16 model using cross validation. 
  * This is due to fact that during Earth test case (#23) I created a neural net with AUC of 0.77 on test set. However it achieved AUC of less than 0.5 on validation set. 
  * Mismatch between AUC on test and val set is due to fact that network was not trained on val set. Cross validation will allow to see neural net behaves on whole dataset 
  * Normally, I'd like to use train/val/test subsets. However, I don't know if this is available in skorch. I will start with train/test set and see what the results are

Expectations
* None. I don't know what to expect from cross validation on neural net. I have never done it

**Comment by Domnyk** [comment]
* Skorch ease of use
Skorch is hard to use with PyTorch. When it comes to using cross validation it is extra hard. There is no way to define separate transforms for folds which are used for training and the one used for testing

I will try to implement cross validation by myself in PyTorch

**Comment by Domnyk** [comment]
**Results**
Cross validation resulted in AUC of 0.5
